src/PlatformSockets.c

```diff
@@ -220,6 +220,10 @@ void closeSocket(SOCKET s) {
 #endif
 }
 
+#ifdef __3DS__
+static const in_port_t valid_udp_ports[] = {47998, 47999, 48000, 48002, 48010};
```

Comment:
Similar to the above comment, I assumed trying each of the ports mentioned in the Setup Guide would be an acceptable way to manually bind to the right port, I could also add a way to specify the port from the function input, or just pass the port through `localAddr`. I'm open to suggestions here.